packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -132,6 +132,35 @@ export type AreParamEqual<A extends (props: any) => JSX.Element | undefined, B e
 // @public
 export type AreTypeEqual<A, B> = A extends B ? (B extends A ? true : false) : false;
 
+// @beta
+export interface AtMentionDisplayOptions {
+    atMentionSuggestionRenderer?: (suggestion: AtMentionSuggestion) => JSX.Element;
```

Comment:
Change the name to onRenderAtMentionSuggestion to align with other callback names

Reply:
Let's think a bit about naming here.
When we see `on<something>`, they should be used for handling events that happen. This is not the role of every callback, however. In the case where we have an `onCancel` event, the cancellation _has happened_ and was triggered by the user.
In the case of this callback, it is actually providing the rendering function and is not called as an event after the rendering has occurred.

For this reason, we should keep `on<something>` to event handler callbacks, rather than just as a blanket naming of callbacks.

Reply:
One of the reason we align with fluent is we reuse lots of their component and most of our contoso customer will use fluent library as well, so it is a lot easier to have same naming convention, we don't want them to fall into different worlds when they are using both. 


Within UI Library internally, for example, there is onRenderContextualItem in fluent, and we wrap it and re-export it as an interface from our side (same name, then we can just spread the object without doing lots of property copy)

To the event problem, there is still ways to differentiate these 2 different cases:
If we wanna say this is happen after the button is rendered, it should be onButtonRendered or onButtonMounted

It's already a convention for a long while in UI library + fluent, so I would say, it would be not ideal to create an exception and have 2 different naming conventions around the UI library